tap_mailchimp/sync.py

```diff
@@ -355,6 +360,12 @@ def get_selected_streams(catalog):
             selected_streams.add(stream.tap_stream_id)
     return list(selected_streams)
 
+def get_selected_fields(catalog, stream_name):
```

Comment:
Should use singer-python library calls to get the metadata and check if a field is selected. Should be put under test as well or commented to explain what its doing.